### PR TITLE
Add Image Component

### PR DESCRIPTION
**Pull request by jvaddinen**
DS-180-A044-Image

Main Reviewer: @AnthonyA44 

Description & Background Context:

This PR includes code for Image Component [Figma](https://www.figma.com/file/YFz6bzcX1DSAQfu4BSfm6G/Design-System-Extended?node-id=622%3A334)

#### Where should the reviewer start?

File name: Image.js 

#### How should this be tested?
On Storybook Link 

#### Are there any breaking changes?
NO

### CODE REVIEW

File: src/components/Image/Image.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+      <img src={src} alt={alt} />
+    </>
+  );
+}
+
+Image.propTypes = {
+  /**
+   * The text that the button will display
+   */
+  id: PropTypes.string,
```
Comment (AnthonyA44) [inline src/components/Image/Image.js:18]: Minor updates needed to the explanation text for each parameter, to explain the individual parameters usage :)

**Review by AnthonyA44 — APPROVED** [review]

File: src/components/Image/Image.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+  src: PropTypes.string.isRequired,
+
+  /**
+   * css overrides for button
+   */
+  className: PropTypes.string,
+  /**
+   * To identify the heading element
+   */
+  id: PropTypes.string.isRequired,
```
Comment (AnthonyA44) [inline src/components/Image/Image.js:37]: duplicate id props

**Review by AnthonyA44 — CHANGES_REQUESTED** [review]
some minor comment changes and duplicate parameters

File: src/components/Image/Image.js (hunk 2 above)
Comment (jvaddinen) [reply src/components/Image/Image.js]: Removed

**Review by AnthonyA44 — APPROVED** [review]
Everything looks good. Component works as intended on external repo :)

**Pull request merged**